app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Please enable `:confirmable`   in the user model for the email confirmation

Reply:
Hello @hiromataba, thanks for the review. I have added the email confirmation. As for the first issue you raised, it is probably due to the fact that we have different ruby versions.